pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/tasks/action/pip/PipWheelAction.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2016 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.gradle.python.tasks.action.pip;
+
+import com.linkedin.gradle.python.extension.PythonDetails;
+import com.linkedin.gradle.python.extension.WheelExtension;
+import com.linkedin.gradle.python.plugin.PythonHelpers;
+import com.linkedin.gradle.python.exception.PipExecutionException;
+import com.linkedin.gradle.python.tasks.exec.ExternalExec;
+import com.linkedin.gradle.python.util.EnvironmentMerger;
+import com.linkedin.gradle.python.util.PackageInfo;
+import com.linkedin.gradle.python.util.PackageSettings;
+import com.linkedin.gradle.python.wheel.WheelCache;
+import org.apache.commons.io.FileUtils;
+import org.gradle.api.Project;
+import org.gradle.api.file.ConfigurableFileTree;
+import org.gradle.api.logging.Logger;
+import org.gradle.api.logging.Logging;
+import org.gradle.process.ExecResult;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+public class PipWheelAction extends AbstractPipAction {
```

Comment:
This class is not used anywhere.
The `PipInstallAction` was refactored from` PipInstallTask` and used there, AFAICS.
OTOH, this class seems to be refactored from `BuildWheelsTask`, but that class was not changed in this PR and this class is not used in it. Omission?

Reply:
Yep. I missed it. Will add w/ next update.